ui/app/templates/components/identity/item-details.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="box is-shadowless is-marginless is-fullwidth">
-  {{#if model.disabled}}
+  {{#if (and model.disabled)}}
```

Comment:
TIL

Reply:
Ha whoops, that was left in from a debug - that's how I found the (has-block) issue - I'll remove this.

Reply:
Updated!